Controller/InternalToolsController.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace Leadin\SurvivalKitBundle\Controller;
+
+use Leadin\SurvivalKitBundle\Logging\LogContext;
+use Leadin\SurvivalKitBundle\Logging\Logger;
+use Leadin\SurvivalKitBundle\HttpHelper\HttpServerHelper\ITokenAuthenticatedController;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+/**
+ * @Route("/internal-tools", name="survival_kit_internal_tools_");
+ */
+class InternalToolsController extends AbstractController implements ITokenAuthenticatedController
```

Comment:
InternalTools does not means any thing ...
could be to have some comment to explain